p4-16/spec/P4-16-spec.mdk

```diff
@@ -2486,7 +2486,7 @@ struct Parsed_headers {
 ### Tuple types { #sec-tuple-types }
 
 A tuple is similar to a `struct`, in that it holds multiple
-values. Unlike a `struct` type, tuples have no named fields. The
+values. The fields of a tuple are named in order `f0`, `f1`, etc.. The
```

Comment:
Should be "etc." not "etc.."

Reply:
Let's see if can implement an alternative prototype for this.

Reply:
> Technically one dot is for the etc abbreviation, and the second is the sentence-ending dot.

That might seem logical, but English defies logic in many ways. https://english.stackexchange.com/questions/8382/when-etc-is-at-the-end-of-a-phrase-do-you-place-a-period-after-it/8385#8385

Reply:
The Chicago Manual of Style agrees :-)

https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation/faq0040.html